packages/apps/human-app/server/src/common/config/environment-config.service.ts

```
@@ -58,6 +59,9 @@ export class EnvironmentConfigService {
       DEFAULT_CACHE_TTL_ORACLE_DISCOVERY,
     );
   }
+  get rpcUrl(): string {
+    return this.configService.get<string>('RPC_URL', '');
```

it shouldn't return an empty value as it may cause bugs that are hard to debug. Let's throw an error here as this is not expected behaviour in any way